server/models/Category.model.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const categorySchema = new mongoose.Schema({
```

Comment:
Categories should also have a userID associated with them as we probably don't want to show every category ever created to every user. Not to mention the various other problems of category spamming affecting all users, inability to delete a personal category because someone else decided to use it, inappropriate categories being shared with all users, etc.

We can definitely have default categories, but otherwise user created categories should have userID associated with them.

Reply:
Fair enough. I will add this